docs/installation/migrate-to-xcp-ng.md

```diff
@@ -249,6 +249,91 @@ qemu-img convert -f vmdk -O vpc myVMwaredisk.vmdk /run/sr-mount/<SR UUID>/`uuidg
 
 You need to rescan the SR where you new VHD file is, so it can be detected. It will appear in the disk list, without a name or description though. Attach it to any VM you created before (eg without booting it first), and boot.
 
+### Troubleshooting
```

Comment:
This section seems to be written with V2V in mind, . Should it be under the V2V section rather than here? Or improved so that it covers all migration scenarios?